tools/ports/harfbuzz.py

```diff
@@ -35,7 +35,7 @@ def process_dependencies(settings):
 def process_args(ports, args, settings, shared):
   if settings.USE_HARFBUZZ == 1:
     get(ports, settings, shared)
-    args += ['-Xclang', '-isystem' + os.path.join(ports.get_build_dir(), 'harfbuzz')]
+    args += ['-Xclang', '-isystem' + os.path.join(ports.get_dir(), 'harfbuzz', 'harfbuzz-' + TAG, 'src')]
```

Comment:
why is this change needed? we should have copied the project into the build dir, so the headers should be there. Perhaps we were just missing the `'src'` addition at the end?

Reply:
Basically, this is what the folder structure looks like when Harfbuzz is extracted:
```
$ find .emscripten_ports -name hb.h
.emscripten_ports/harfbuzz/harfbuzz-1.7.5/src/hb.h
```
I'm not familiar with how ports works. I basically just looked at #6273, and @ebraminio asked me to take over :P  What would be the correct way to copy the headers over? Note that Harfbuzz doesn't have an `include` folder. Everything is in `src`.

Reply:
I've added a `make install` step there. I hope it's the right to call `make()` twice. Looks a bit weird to me since I don't really know what it does...